### ISSUE TITLE
`Cannot promisify an API that has normal methods with 'Async'-suffix` when using with Claudia and Nuxt

### ISSUE THREAD

**Issue by nkoder** [jantimon/html-webpack-plugin#654]
**tl;dr** I have a problem and I can create not invasive (probably ;-) ) easy-fix for that (will create PR in a moment)

It's hard for me to describe my full setup but basics are:
* macOS 10.12.4
* node 6.10.1 (and same on 7.7.4)
* npm 4.5.0
* [claudia]( https://claudiajs.com/ ) 2.11.1
* [nuxt]( https://nuxtjs.org/ ) 0.10.6 or any > 0.9.9 (error was introduced with added dependency on `html-webpack-plugin` 2.28.0)

When I want to deploy my updated source code of Nuxt-bassed project to AWS Lambda with Claudia, I get following error from `claudia update` subcommand:
```
$ claudia update --use-local-dependencies --version testing --set-env NODE_ENV=production,BASE_PATH='',BASE_URL='https://testing.timbercode.pl'
validating package
TypeError: Cannot promisify an API that has normal methods with 'Async'-suffix

    See http://goo.gl/MqrFmX

    at checkValid (/private/var/folders/c5/s701bk894ld2yyxjy6hvshqm0000gn/T/649cc050-bbff-495b-8752-ffc800e9a8a0/node_modules/bluebird/js/release/promisify.js:54:27)
    at promisifiableMethods (/private/var/folders/c5/s701bk894ld2yyxjy6hvshqm0000gn/T/649cc050-bbff-495b-8752-ffc800e9a8a0/node_modules/bluebird/js/release/promisify.js:77:5)
    at promisifyAll (/private/var/folders/c5/s701bk894ld2yyxjy6hvshqm0000gn/T/649cc050-bbff-495b-8752-ffc800e9a8a0/node_modules/bluebird/js/release/promisify.js:241:9)
    at Function.Promise.promisifyAll (/private/var/folders/c5/s701bk894ld2yyxjy6hvshqm0000gn/T/649cc050-bbff-495b-8752-ffc800e9a8a0/node_modules/bluebird/js/release/promisify.js:311:12)
    at Object.<anonymous> (/private/var/folders/c5/s701bk894ld2yyxjy6hvshqm0000gn/T/649cc050-bbff-495b-8752-ffc800e9a8a0/node_modules/html-webpack-plugin/index.js:10:9)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/private/var/folders/c5/s701bk894ld2yyxjy6hvshqm0000gn/T/649cc050-bbff-495b-8752-ffc800e9a8a0/node_modules/nuxt/dist/nuxt.js:2173:18)
    at __webpack_require__ (/private/var/folders/c5/s701bk894ld2yyxjy6hvshqm0000gn/T/649cc050-bbff-495b-8752-ffc800e9a8a0/node_modules/nuxt/dist/nuxt.js:21:30)
    at Object.<anonymous> (/private/var/folders/c5/s701bk894ld2yyxjy6hvshqm0000gn/T/649cc050-bbff-495b-8752-ffc800e9a8a0/node_modules/nuxt/dist/nuxt.js:1700:78)
    at __webpack_require__ (/private/var/folders/c5/s701bk894ld2yyxjy6hvshqm0000gn/T/649cc050-bbff-495b-8752-ffc800e9a8a0/node_modules/nuxt/dist/nuxt.js:21:30)
cannot require ./lambda after clean installation. Check your dependencies.
error Command failed with exit code 1.
```

It appears that newer Nuxt is using `html-webpack-plugin` 2.28.0, which calls `Promisify.all(fs)`, which turns to throw exception described above (at least in my configuration). The easy-fix solution would be to provide different suffix for `Promisify.all` to not use default `Assync` suffix. Then I would make PR to Nuxt in order to update `html-webpack-plugin` and my problem would be solved. **I've checked that my fix works by edition of source code of locally edited dependencies in `node_modules`** :-)

**Comment by nkoder** [comment]
It looks like I was too quick to write an issue... I've found a more real reason for my problem in another library :-/

I will investigate more and then will delete/close/update this issue.

**Comment by nkoder** [comment]
OK, so I've created a fix in more apriopriate place: https://github.com/claudiajs/claudia/pull/118
